style/components/tasklist.css

```diff
@@ -0,0 +1,5 @@
+.tasklist {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
```

Comment:
did you create entire file just for doing flex things?

Reply:
PS: you created this file because we discussed you have to. But you forgot why you had to create it and this shit happened. Use this file correctly.